packages/DEBIAN/templates

```diff
@@ -0,0 +1,5 @@
+Template: vufind/solr-user
+Type: string
+Default: vufind-solr
+Description: User name for Solr:
+ Please provide an account name for a user to run VuFind's Solr.
```

Comment:
Can we sneak in the ® symbol here, or will it cause encoding/display problems?

```suggestion
 Please provide an account name for a user to run VuFind®'s Solr.
```

Reply:
I'll check!

Reply:
It seems to work fine. :)